openapi.yaml

```diff
@@ -269,6 +269,91 @@ paths:
               schema:
                 $ref: '#/components/schemas/Problem'
 
+  /share:
+    post:
+      summary: Save the given AsyncAPI document.
+      operationId: saveDoc
+      tags:
+        - share
+      requestBody:
+        description: Parse your AsyncAPI document.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ShareRequest"
+      responses:
+        "200":
+          description: AsyncAPI document successfully saved.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ShareResponse"
+        "400":
+          description: The given AsyncAPI document(s) is/are not valid.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "422":
+          description: The given AsyncAPI document(s) is/are not valid due to invalid parameters in the request.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        default:
+          description: Unexpected problem.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+  /share/{id}:
+    get:
+      summary: Retrieve a given AsyncAPI document
+      operationId: getDoc
+      tags:
+        - share
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema: 
+            $ref: "#/components/schemas/RetrieveRequest" 
+      schemas:
+        type: string
+      requestBody:
+        description: AsyncAPI document successfully saved.
```

Comment:
It's not a valid description for this "request body" - even we don't need that, because it's a normal `get`, not `post` :)